icons/webgpu/webgpu-original-wordmark.svg

```diff
@@ -0,0 +1 @@
http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path fill="#0086e8" stroke="#0086e8" stroke-width=".18" d="M108.423 49.459 97.589 30.695h21.698z"/><path fill="#0093ff" stroke="#0093ff" stroke-width=".18" d="M108.423 11.932 97.589 30.695h21.698z"/><path fill="#0076cc" stroke="#0076cc" stroke-width=".18" d="M86.758 86.986 65.09 49.46h43.334Z"/><path fill="#0066b0" stroke="#0066b0" stroke-width=".18" d="M86.758 11.931 65.09 49.46h43.334z"/><path fill="#005a9c" stroke="#005a9c" stroke-width=".18" d="M43.424 86.986.09 11.931h86.666Z"/><path d="m25.988 94.81-4.626 17.809-4.837-17.809H13.44l-5.08 17.87-4.595-17.87H.832l5.896 21.104 3.054-.03 5.08-16.962 4.897 16.962h3.084l6.108-21.074zm21.104 12.155a8.346 8.346 0 0 0-.983-4.067 7.122 7.122 0 0 0-2.797-2.827 8.318 8.318 0 0 0-4.172-1.028 8.595 8.595 0 0 0-4.278 1.044 7.216 7.216 0 0 0-2.888 2.979q-1.028 1.935-1.028 4.505t1.059 4.52a7.487 7.487 0 0 0 2.917 3.008 8.37 8.37 0 0 0 4.218 1.058q2.963 0 4.959-1.496a7.124 7.124 0 0 0 2.66-3.885h-2.963a4.402 4.402 0 0 1-1.678 2.222q-1.194.832-2.978.831a5.335 5.335 0 0 1-3.704-1.375q-1.527-1.376-1.678-3.825h13.243a16.235 16.235 0 0 0 .09-1.663zm-2.842-.575H33.788q.212-2.358 1.678-3.688a5.1 5.1 0 0 1 3.553-1.33 5.894 5.894 0 0 1 2.646.589 4.525 4.525 0 0 1 1.89 1.723 5.078 5.078 0 0 1 .695 2.706zm9.221-3.99v-8.89H50.72v22.374h2.751v-3.054a6.372 6.372 0 0 0 2.434 2.389 7.37 7.37 0 0 0 3.764.937 7.537 7.537 0 0 0 6.834-4.142 9.512 9.512 0 0 0 1.028-4.475q0-2.54-1.028-4.46a7.498 7.498 0 0 0-2.813-2.978 7.733 7.733 0 0 0-4.02-1.058 7.338 7.338 0 0 0-3.72.937 6.507 6.507 0 0 0-2.48 2.419zm11.248 5.14a6.896 6.896 0 0 1-.756 3.31 5.316 5.316 0 0 1-2.041 2.147 5.558 5.558 0 0 1-2.827.74 5.5 5.5 0 0 1-2.797-.74 5.414 5.414 0 0 1-2.056-2.147 7.3 7.3 0 0 1 0-6.53 5.418 5.418 0 0 1 2.056-2.147 5.503 5.503 0 0 1 2.797-.741 5.65 5.65 0 0 1 2.827.725 5.238 5.238 0 0 1 2.041 2.117 6.74 6.74 0 0 1 .756 3.265zm25.7-6.623a9.517 9.517 0 0 0-3.765-4.686q-2.585-1.663-5.94-1.663a10.637 10.637 0 0 0-5.353 1.376 10.073 10.073 0 0 0-3.824 3.84 11.62 11.62 0 0 0 0 11.08 10.099 10.099 0 0 0 3.824 3.825 10.632 10.632 0 0 0 5.352 1.376 10.251 10.251 0 0 0 5.05-1.255 10.086 10.086 0 0 0 3.658-3.416 10.466 10.466 0 0 0 1.633-4.762v-2.116H79.715v2.237h8.406q-.333 3.146-2.328 4.989-1.995 1.843-5.08 1.844a7.854 7.854 0 0 1-3.976-1.013 7.176 7.176 0 0 1-2.781-2.887q-1.013-1.875-1.013-4.354 0-2.48 1.013-4.369a7.159 7.159 0 0 1 2.781-2.903 7.858 7.858 0 0 1 3.976-1.012 7.293 7.293 0 0 1 3.87 1.012 6.559 6.559 0 0 1 2.54 2.857zm18.352.06q0-2.72-1.86-4.444-1.858-1.723-5.426-1.723H94.68v21.074h2.752v-8.798h4.051q3.689 0 5.488-1.739 1.799-1.738 1.8-4.369zm-7.286 3.84h-4.052v-7.739h4.052q4.474 0 4.474 3.9 0 1.845-1.088 2.843-1.088.997-3.386.997zm10.642-10.007v13.304q0 2.63 1.044 4.43a6.548 6.548 0 0 0 2.842 2.675 9.728 9.728 0 0 0 8.073 0 6.644 6.644 0 0 0 2.857-2.676q1.057-1.798 1.057-4.43V94.81h-2.75v13.334q0 2.812-1.377 4.172-1.375 1.36-3.794 1.36-2.449 0-3.825-1.36t-1.375-4.172V94.81h-2.752z"/></svg>
```

Comment:
There are some stroke lines in this icon. Here goes a suggestion without any of that.
<img width="256" src="https://github.com/devicons/devicon/assets/76992016/6c67b271-a171-4f8d-bca8-7ef23abf9162">

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path fill="#0086e8" d="m97.434 30.605 10.99 19.034.076-.135 10.943-18.899h-22.01z"/><path fill="#0093ff" d="m108.424 11.752-.078.135-10.912 18.898h22.01l-11.02-19.033z"/><path fill="#0076cc" d="m64.934 49.37 21.824 37.796.078-.135L108.58 49.37H64.934z"/><path fill="#0066b0" d="m86.758 11.752-.078.135L64.934 49.55h43.646L86.758 11.752z"/><path fill="#005a9c" d="m-.066 11.842 43.49 75.324.078-.135 43.41-75.19H-.066z"/><path d="m25.988 94.81-4.626 17.809-4.837-17.809H13.44l-5.08 17.87-4.595-17.87H.832l5.896 21.104 3.054-.03 5.08-16.962 4.897 16.962h3.084l6.108-21.074zm21.104 12.155a8.346 8.346 0 0 0-.983-4.067 7.122 7.122 0 0 0-2.797-2.827 8.318 8.318 0 0 0-4.172-1.028 8.595 8.595 0 0 0-4.278 1.044 7.216 7.216 0 0 0-2.888 2.979q-1.028 1.935-1.028 4.505t1.059 4.52a7.487 7.487 0 0 0 2.917 3.008 8.37 8.37 0 0 0 4.218 1.058q2.963 0 4.959-1.496a7.124 7.124 0 0 0 2.66-3.885h-2.963a4.402 4.402 0 0 1-1.678 2.222q-1.194.832-2.978.831a5.335 5.335 0 0 1-3.704-1.375q-1.527-1.376-1.678-3.825h13.243a16.235 16.235 0 0 0 .09-1.663zm-2.842-.575H33.788q.212-2.358 1.678-3.688a5.1 5.1 0 0 1 3.553-1.33 5.894 5.894 0 0 1 2.646.589 4.525 4.525 0 0 1 1.89 1.723 5.078 5.078 0 0 1 .695 2.706zm9.221-3.99v-8.89H50.72v22.374h2.751v-3.054a6.372 6.372 0 0 0 2.434 2.389 7.37 7.37 0 0 0 3.764.937 7.537 7.537 0 0 0 6.834-4.142 9.512 9.512 0 0 0 1.028-4.475q0-2.54-1.028-4.46a7.498 7.498 0 0 0-2.813-2.978 7.733 7.733 0 0 0-4.02-1.058 7.338 7.338 0 0 0-3.72.937 6.507 6.507 0 0 0-2.48 2.419zm11.248 5.14a6.896 6.896 0 0 1-.756 3.31 5.316 5.316 0 0 1-2.041 2.147 5.558 5.558 0 0 1-2.827.74 5.5 5.5 0 0 1-2.797-.74 5.414 5.414 0 0 1-2.056-2.147 7.3 7.3 0 0 1 0-6.53 5.418 5.418 0 0 1 2.056-2.147 5.503 5.503 0 0 1 2.797-.741 5.65 5.65 0 0 1 2.827.725 5.238 5.238 0 0 1 2.041 2.117 6.74 6.74 0 0 1 .756 3.265zm25.7-6.623a9.517 9.517 0 0 0-3.765-4.686q-2.585-1.663-5.94-1.663a10.637 10.637 0 0 0-5.353 1.376 10.073 10.073 0 0 0-3.824 3.84 11.62 11.62 0 0 0 0 11.08 10.099 10.099 0 0 0 3.824 3.825 10.632 10.632 0 0 0 5.352 1.376 10.251 10.251 0 0 0 5.05-1.255 10.086 10.086 0 0 0 3.658-3.416 10.466 10.466 0 0 0 1.633-4.762v-2.116H79.715v2.237h8.406q-.333 3.146-2.328 4.989-1.995 1.843-5.08 1.844a7.854 7.854 0 0 1-3.976-1.013 7.176 7.176 0 0 1-2.781-2.887q-1.013-1.875-1.013-4.354 0-2.48 1.013-4.369a7.159 7.159 0 0 1 2.781-2.903 7.858 7.858 0 0 1 3.976-1.012 7.293 7.293 0 0 1 3.87 1.012 6.559 6.559 0 0 1 2.54 2.857zm18.352.06q0-2.72-1.86-4.444-1.858-1.723-5.426-1.723H94.68v21.074h2.752v-8.798h4.051q3.689 0 5.488-1.739 1.799-1.738 1.8-4.369zm-7.286 3.84h-4.052v-7.739h4.052q4.474 0 4.474 3.9 0 1.845-1.088 2.843-1.088.997-3.386.997zm10.642-10.007v13.304q0 2.63 1.044 4.43a6.548 6.548 0 0 0 2.842 2.675 9.728 9.728 0 0 0 8.073 0 6.644 6.644 0 0 0 2.857-2.676q1.057-1.798 1.057-4.43V94.81h-2.75v13.334q0 2.812-1.377 4.172-1.375 1.36-3.794 1.36-2.449 0-3.825-1.36t-1.375-4.172V94.81h-2.752z"/></svg>

```